Monika After Story/game/definitions.rpy

```diff
@@ -2959,6 +2959,61 @@ init -1 python:
     def mas_isMonikaBirthday():
         return datetime.date.today() == mas_monika_birthday
 
+    def mas_isSpring():
+        """
+        Checks if it's spring right now
+
+        Note: If persistent._mas_pm_live_north_hemisphere is none, we assume northern hemi
+
+        RETURNS:
+            boolean showing whether or not it's spring right now
+        """
+        if persistent._mas_pm_live_north_hemisphere is not False:
+            return ((datetime.date(datetime.date.today().year,3,21) <= datetime.date.today()) and (datetime.date.today() < datetime.date(datetime.date.today().year,6,21)))
```

Comment:
can you make all of these specific dates (3-21, 6-21, 9-23, 12-21) variables, similar to how o31 and 9-22 are? give them appropriate names, too.